rusoto/core/src/lib.rs

```diff
@@ -31,19 +31,14 @@ extern crate lazy_static;
 #[macro_use]
 extern crate log;
 extern crate base64;
```

Comment:
technically some of these other extern crates can go away because this crate uses 2018 edition but I wanted to limit scope